README.md

```diff
@@ -69,15 +72,31 @@ const messageBytes = Buffer.from("example 2");
 expect( Codec.Signature.encode(messageBytes) ).to.equal(base64String);
 
 
-const hashString = "QmNZAJfVYoCASiPc3uYZXrvhRFbxJLxG18R2Ga4ZXfP4kR";
-const hashBytes = await sha256(new Uint8Array([0xca, 0xfe]));
+const holoHashType = "entry";
+const hashString = "uhCEkWCsAgoKkkfwyJAglj30xX_GLLV-3BXuFy436a2SqpcEwyBzm";
+const dataBytes					= new Uint8Array([
+  88,	 43,  0,	 130,	130, 164, 145, 252,
+  50,	 36,  8,	 37,	143, 125, 49,	 95,
+  241, 139, 45,	 95,	183, 5,		123, 133,	
+  203, 141,	250, 107,	100, 170, 165, 193
+]);
```

Comment:
Good idea.  Since we now primarily use base64, I think it'd be best to display the string in the base64 as well rather than hex.  Otherwise, I'm on board.